src/generic/YoungTabs.jl

```diff
@@ -22,38 +22,15 @@ julia> p = Partition([4,3,1]); size(p)
 """
 size(p::Partition) = size(p.part)
 
-Base.IndexStyle(::Type{Partition}) = Base.IndexLinear()
-
 @doc Markdown.doc"""
     getindex(p::Partition, i::Integer)
 
-Return the `i`-th part (in decreasing order) of the partition.
+Return the `i`-th part (in non-increasing order) of the partition.
 """
 getindex(p::Partition, i::Integer) = p.part[i]
 
 Base.sum(p::Partition) = p.n
 
-@doc Markdown.doc"""
```

Comment:
Am I correct that this functionality is being removed and not replaced? In that sense the PR should be marked breaking, right?

Reply:
yes, I mentioned in the description (in bold :D) that this pull is breaking ;) feel free to add a label (I don't have permissions)

Reply:
Thanks. I didn't know you didn't have permissions. I've added the breaking label.